exercises/isbn-verifier/canonical-data.json

```diff
@@ -151,9 +151,8 @@
         },
         {
             "uuid": "daad3e58-ce00-4395-8a8e-e3eded1cdc86",
-            "description": "invalid characters are not ignored",
+            "description": "invalid characters are not ignored before checking length",
             "comments": ["Catch invalid characters in an otherwise valid isbn"],
```

Comment:
Now that the description will be updated, does the comment provide any additional information?

Reply:
Hmm, there may still be some value in noting that the ISBN was otherwise valid! Or one might argue that "in an otherwise-valid ISBN" should be moved to the description instead? Might make it a bit too long though. Might leave it as-is for that reason.

Reply:
Okay, just checking.